frontend/src/main/components/Courses/SchoolDropdown.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+import {useFormContext} from "react-hook-form";
+
+const SchoolDropdown = ({schools = [], testId}) => {
+    const {
+        register,
+        formState: {errors}
+    } = useFormContext();
+
+    return (<Form.Group>
+        <Form.Label htmlFor="school">School</Form.Label>
+        <Form.Control data-testid={`${testId}-school`} id="school"  as="select"
+          isInvalid={Boolean(errors.school)}
+          {...register("school", {required: true, minLength: 2})}
+        >
+            <option value=""></option>
+            {schools.map(function (object, i) {
+                const key = `${testId}-option-${i}`
+                return(
+                    <option key={key} data-testid={key} value={object.name}>
+                        {object.name}
+                    </option>

```

Comment:
I think I would have expected you to do something with `initialContents` since you are passing it in here in the storybook.
```
initialContents: {"school":"ucsb"},
```

Maybe you intended some code to make "ucsb" the preselected option?

This might help (or not): 
* https://stackoverflow.com/questions/41174974/specify-default-value-to-formcontrol-of-react-bootstrap
* https://www.geeksforgeeks.org/react-bootstrap-select/

Reply:
Hey professor! Because the SchoolDropdown requires a form surrounding it, there's a model form at the bottom of the Storybook file where initialContents is used and passed into defaultValues that SchoolDropdown inherits. I did forget to change it to the value though, which is why it didn't show up